saja_lahaleeh.py

```diff
@@ -0,0 +1,39 @@
+
+#T1
```

Comment:
Please make the comments clear as mentioned in the [FIRST-NAME_LAST-NAME.py ](https://github.com/AHL-Logics-backend-tranining-2024/python-task-1/blob/main/FIRST-NAME_LAST-NAME.py) file.

Example:
`# Step 1: Initialize Book Inventory`
`# Initialize a list of books, each represented as a dictionary with title, author, availability, and tags.`